source/common/grpc/json_transcoder_filter.h

```diff
@@ -62,6 +62,14 @@ class JsonTranscoderConfig : public Logger::Loggable<Logger::Id::config> {
                    std::unique_ptr<google::grpc::transcoding::Transcoder>& transcoder,
                    const Protobuf::MethodDescriptor*& method_descriptor);
 
+
```

Comment:
This file and the `.cc` are out-of-sync (the newly declared field doesn't match). Can you update? Thanks?